tests/group_functions/group_barrier.cpp

```diff
@@ -149,11 +149,17 @@ TEMPLATE_TEST_CASE_SIG("Group barriers", "[group_func][dim]", ((int D), D), 1,
       local_acc[llid] = 0;
       sycl::group_barrier(group);
 
+      if (local_acc[max_id - llid] != 0)
+        std::get<s::test>(group_barriers_acc[0]) = false;
+      // make sure we check all items before moving on
+      sycl::group_barrier(group);
```

Comment:
I have been puzzled a little bit by this one.
If the barrier does not work at the first place we will have UB since we will read some non initialized values. But anyway, if there is the barrier not working, everything is UB at the first place, so, not a big deal. ;-)
Furthermore, if it happens that the implementation initialize the local accessor to 0, this might be unnoticed.
So what about for example writing `53 + llid` instead of 0 on line 149 and update accordingly line 152?

Reply:
Done. Changed to `llid` for initializations.